src/main/java/application/rest/v1/configmaps/KindActionMappingProcessor.java

```diff
@@ -0,0 +1,462 @@
+/*
```

Comment:
I request you document test cases and provide sample KAMs for those test cases.  There needs to be someway to execute those tests to confirm correct behavior.  Store this all somewhere in Git.